ramips: add missing LEDs and modem control for ASUS 4G-AX56

mrhaav opened the pull request:
Add missing LEDs and modem control for ASUS 4G-AX56
 - wifi2.4G white
 - wifi5G   white
 - wan      two-coloured, white and red
 - modem    four-coloured white, blue, yellow and red
 - rssi-1   white
 - rssi-2   white
 - rssi-3   white
	
and modem reboot and reset

Signed-off-by: Henrik Ginstmark <henrik@ginstmark.se>

namiltd commented on target/linux/ramips/dts/mt7621_asus_4g-ax56.dts:
```diff
@@ -31,6 +31,67 @@
+		};
+
+		modem_red {
+			label = "red:modem";
```
use:
color = <LED_COLOR_ID_RED>;
function = LED_FUNCTION_MOBILE;

namiltd commented:
Combine it into one commit
```
git reset --hard c9ea2799b7a8f56c53a37406b1a8f22fd9817f62
git cherry-pick -n 73e3d0a2cb39b67091f9ae10c15afeda79249c38
git cherry-pick -n 7e5ff4d2806ef1c8342fa35d09d268ccac6fce06
git commit 
git pull --force
```

mrhaav committed:
ramips: add missing LEDs and modem control for ASUS 4G-AX56

Add missing LEDs and modem control for ASUS 4G-AX56
 - wifi2.4G white
 - wifi5G   white
 - wan      two-coloured, white and red
 - modem    four-coloured white, blue, yellow and red
   change
      label = "xxxx:modem";
   to
      color = <LED_COLOR_ID_xxxx>;
      function = LED_FUNCTION_MOBILE;
 - rssi-1   white
 - rssi-2   white
 - rssi-3   white

and modem reboot and reset

Combined into one commit

Signed-off-by: Henrik Ginstmark <henrik@ginstmark.se>
Link: https://github.com/openwrt/openwrt/pull/17927
Signed-off-by: Hauke Mehrtens <hauke@hauke-m.de>

DragonBluep commented:
Please reset the author name `mrhaav <henrik@ginstmark.se>` to fix the CI warning.

mrhaav commented:
> Please reset the author name `mrhaav <henrik@ginstmark.se>` to fix the CI warning.

Hi
How do I reset the author name?
When I created the PR I had not included my full name in my github profile. When I saw the CI warning I added my full name.
Is it possible to re-run the check or do I need to do something more?

namiltd commented:
```
git commit --amend --author="Henrik Ginstmark <henrik@ginstmark.se>" --no-edit
git push --force
```
